### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by AetherPrior**
Used reactjs-express-generator to make initial file structure to work on

### CODE REVIEW

**Comment by Kushagra-0801** [comment]
Are the react files supposed to have `.jsx` extension?

**Comment by AetherPrior** [comment]
The setup added them by default. I don't think it is a necessity

**Review by Kushagra-0801 — APPROVED** [review]

File: src/app/components/User.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+class User extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+        name: ""
+      }
+    };
+    this.fetchUsers = this.fetchUsers.bind(this);
+  }
```
Comment (VikramjeetD) [inline src/app/components/User.jsx:12]: Isn't user more of a model than a React component? We're not "displaying" it for the front end per se.
For later, we probably need this to be a mongoose schema (for MongoDB) for full signup functionality.

Comment (AetherPrior) [reply src/app/components/User.jsx:12]: Yes. we'll need this to be a schema than a react component. I'll remove this change for now

**Review by Kushagra-0801 — COMMENTED** [review]
Why did you change the license?

**Pull request merged**